main.js

```diff
@@ -1,32 +1,47 @@
-
-
-let str = "https://github.com/?username=6thSence";
+let url = window.location.toString();
+console.log(url);
+//let str = "https://github.com/?username=6thSence";
 let nickName = (link) => {
   let arr = link.split('=');
-  return (arr[arr.length-1]);
+  console.log(arr);
+  let nick = arr[1];
+  console.log(nick);
+  if(nick==undefined) {
+    nick = "x1nax";
+  }
+  return nick;
 }
-nickName(str);
+nickName(url);
+console.log(nickName(url));
+
 
 let info = (user) => {
   fetch('https://api.github.com/users/'+ user)
-    .then(res => res.json())
-      .then(res => {
-        let newImg = document.createElement("IMG");
-        let newh1 = document.createElement("h1");
-        let newp = document.createElement("p");
-        let newA = document.createElement("a");
-        newImg.src = res.avatar_url;
-        newh1.innerHTML = res.name;
-        newp.innerHTML = res.bio;
-        newA.href = res.html_url;
-        document.body.appendChild(newA);
-        newA.appendChild(newh1);
-        document.body.appendChild(newp);
-        document.body.appendChild(newImg);
-
-        }
-      );
-
+  .then(res => {
+    if (res.status != 404) {
```

Comment:
Codacy found an issue: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/gh/WebHeroSchool/jstask35/pullRequest?prid=6095927)